src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,218 @@
+        description={`Read about Material Design's navigation fundamentals.`}
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+    <Divider />
+</React.Fragment>
+
+# Drawer Navigation
+
+A navigation drawer is a panel that is attached on the side of your application. These typically contain a [List](/patterns/lists) of primary pages in your application for the user to access quickly.
+
+<ImageGrid images={[NavigationBreakdownImg]} regularWidth />
+
+Most drawers have four main sections:
+
+-   **Header**: contains the name of the company
+-   **Subheader**: contains the product 
+-   **Body**: contains a list of primary pages that users can navigate too
+-   **Footer**: contains company logo, version number and trademark information
+
+There are several different styles of drawer navigation, each with slightly different behavior.
+
+<DemoCard
+    repository={'Drawer'}
+    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
+    react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
 />
 
-## Fixed Navigation
+## Permanent
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
 
-## Collapsible Navigation
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
 
-A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
+<ImageGrid images={[NavigationPermanentImg]} regularWidth />
 
-### Content-Resize / Content-Push
+## Rail
 
-<ImageGrid images={[PermanentImg]} />
+A navigation rail is a permanently collapsed drawer. The rail is able to show less information than a standard drawer, but has the advantage of a more compact, space-saving package. Use this variation if you have:
 
-The content-resize implementation does not cover any of the main application content. Rather, the navigation panel slides in and the main application container resizes to accommodate (the navigation panel and main panel are side by side instead of one on top of the other).
+-   fewer than seven top-level pages
+-   a flat navigation hierarchy
+-   limited space
 
-### Content-Cover
+<ImageGrid images={[NavigationRailImg]} regularWidth />
```

Comment:
The nav rail image is 56 px wide with label, inconsistent with our design specs. 

There are only two types of rails: 
- 72px wide, with an optional label text. The label has to be the `caption` typography. The label can hyphenate to a new line, but should never end with an ellipsis to clip off the text. When the label is present, the divider is required.
- 56px wide, with an optional tooltip on hover. No dividers on this one.

And there should usually be a blue 500 opacity 5% highlight background